nvme_metrics.sh

```diff
@@ -50,7 +50,7 @@ device_list="$(nvme list | awk '/^\/dev/{print $1}')"
 # Loop through the NVMe devices
 for device in ${device_list}; do
   json_check="$(nvme smart-log -o json "${device}")"
-  disk="$(echo "${device}" | cut -c6-10)"
+  disk="$(echo "${device}" | grep -Eo 'nvme[0-9]+')"
```

Comment:
You could also just specify the field delimiter to `cut`
```suggestion
  disk="$(echo "${device}" | cut -d/ -f3)"
```
Proof:
```
$ echo /dev/nvme0n1 | cut -d/ -f3
nvme0n1
```
Otherwise, if we're not concerned about "wild" dependencies, the _right_ way to do this is with `basename`:
```
$ basename /dev/nvme0n1
nvme0n1
```
Since `basename` is part of the GNU coreutils suite of tools, I would be _highly_ surprised if it is not present on a system.